frontend/tests/acceptance/create-person-test.js

```diff
@@ -0,0 +1,157 @@
+import { module, skip, test } from "qunit";
+import page from "frontend/tests/pages/people-new";
+import { Interactor as Pikaday } from "ember-pikaday/test-support";
+import { click, currentURL } from "@ember/test-helpers";
+import setupApplicationTest from "frontend/tests/helpers/setup-application-test";
+import { selectChoose } from "ember-power-select/test-support";
+import Notify from "ember-notify";
+import { setLocale } from "ember-intl/test-support";
+
+module("Acceptance | create person", function(hooks) {
+  setupApplicationTest(hooks);
+
+  const notifyStub = Notify.extend({
+    alert(message, options) {
+      options.closeAfter = null;
+      return this.show("alert", message, options);
+    }
+  });
+  /// Skip this test since there is a bug wtih the @renderInPlace attribute  of the powerselect
```

Comment:
Did you create some sort of follow-up for this?

Reply:
https://github.com/orgs/puzzle/projects/5/views/1?pane=issue&itemId=27710056